main.go

```diff
@@ -19,19 +19,21 @@ var (
 	tick     time.Duration
 	warn     time.Duration
 	critical time.Duration
+	device   string
 	report   bool
 )
 
 func main() {
 
-	flag.DurationVar(&tick, "tick", 10*time.Second, "Update rate")
-	flag.DurationVar(&warn, "warn", 20*time.Minute, "Time to start warning. (Warn)")
-	flag.DurationVar(&critical, "critical", 10*time.Minute, "Time to start warning. (Critical)")
-	flag.BoolVar(&report, "report", false, "Print out updates to stdout.")
+	flag.DurationVar(&tick,     "tick",     10*time.Second,                                 "Update rate")
+	flag.DurationVar(&warn,     "warn",     20*time.Minute,                                 "Time to start warning. (Warn)")
+	flag.DurationVar(&critical, "critical", 10*time.Minute,                                 "Time to start warning. (Critical)")
+	flag.StringVar(  &device,   "device",   "/org/freedesktop/UPower/devices/battery_BAT0", "DBus address of the battery device")
```

Comment:
Actually, the device could be `DisplayDevice` and then you can remove the TODO from [upower/upower.go#L6](https://github.com/omeid/upower-notify/blob/master/upower/upower.go#L6). This may actually address #1 and you wouldn't even need to list your device explicitly.